public/app/plugins/datasource/cloudwatch/query-runner/CloudWatchMetricsQueryRunner.test.ts

```diff
@@ -715,6 +716,32 @@ describe('CloudWatchMetricsQueryRunner', () => {
         period: '',
         sqlExpression: '',
         dimensions: { InstanceId: ['foo'] },
+        expression: '',
+      });
+    });
+    it('interpolates values correctly', () => {
```

Comment:
so we have 2 tests for "interpolateMetricsQueryVariables":
- it interpolates dimensions correctly
- and it interpolates values correctly (which also seems to be testing dimensions interpolations)

I might recommend either combining these into one or creating a test for each value field you hope to be able to interpolate you know?